server/services/store/mattermostauthlayer/mattermostauthlayer.go

```diff
@@ -40,23 +42,37 @@ func (pe NotSupportedError) Error() string {
 	return pe.msg
 }
 
+var systemsBot = &mmModel.Bot{
+	Username:    mmModel.BotSystemBotUsername,
+	DisplayName: "System",
```

Comment:
Should this be defined somewhere to ensure match on channels side. Like Username is?

Reply:
Calls and Channels are going to create the same bot in their respective codebases anyways, so yeah ideally the bot definition and its creation should have been in channels, but it was turning out to be a slightly longer task.

I'll create a card to track any cleanup or improvement tasks for this. Would this be fine?

Reply:
Done - https://community-daily.mattermost.com/boards/workspace/qgsck6cts3fwpqwyjiupjm5cde/b4ctmm917q3rzdfi451qtuxjzcw/vd89yph84ajbkufkbgts9m3sb5a/cpix8aqou7bde9manedyxa9nphh